### DISCUSSION TITLE
[QUESTION] How we can manage snapshots in longhorn - Replica got stuck and volumes going to degrade mode

### DISCUSSION THREAD

**Discussion by dseralathan** [longhorn/longhorn#5735] [Q&A]
## Question
> Suggest to use https://github.com/longhorn/longhorn/discussions to ask questions.

## Environment Harvester v1.1.1

 - Longhorn version:v1.3.2
 - Kubernetes version:v1.24.7+rke2r1
 - Node config 
   - OS type and version Harvester v1.1.1
   - CPU per node: 96 C
   - Memory per node: 1TB
   - Disk type SSD
   - Network bandwidth and latency between the nodes:  10 G on same VLAN
 - Underlying Infrastructure (e.g. on AWS/GCE, EKS/GKE, VMWare/KVM, Baremetal):Baremetal

## Additional context

We have a harvester cluster with 10 nodes . Every day some volumes going to degrade mode.. We are suspecting  It was not able to create replica  on nodes and delete the old snaps . We would like to to know how we can manage snapshot deletion time interval for each volume and we don't need a bulk delete at a time .

**Comment by mantissahz** [comment]
@dseralathan Could you create and provide the support bundle when volumes go to degrade mode?
Here a briefly introduction of how snapshots work in Longhorn. https://longhorn.io/docs/1.3.2/concepts/#24-snapshots

**Comment by dseralathan** [comment]
Thanks mantissahz
